exporters/otlp/src/otlp_metric_utils.cc

```diff
@@ -60,21 +60,25 @@ void OtlpMetricUtils::ConvertSumMetric(const metric_sdk::MetricData &metric_data
     proto::metrics::v1::NumberDataPoint *proto_sum_point_data = sum->add_data_points();
     proto_sum_point_data->set_start_time_unix_nano(start_ts);
     proto_sum_point_data->set_time_unix_nano(ts);
-    auto sum_data = nostd::get<sdk::metrics::SumPointData>(point_data_with_attributes.point_data);
-
-    if ((nostd::holds_alternative<int64_t>(sum_data.value_)))
+    if ((nostd::holds_alternative<sdk::metrics::SumPointData>(
+            point_data_with_attributes.point_data)))
```

Comment:
Just curious, when are you getting this error. As `ConvertSumMetrics()` is called for `metric_sdk::AggregationType::kSum` aggregation, and is always supposed to contain `SumPointData`. If you are getting this error, there may be problem somewhere else.